lib/multicity_input.dart

```diff
@@ -11,36 +11,13 @@ class MulticityInput extends StatefulWidget {
 class MulticityInputState extends State<MulticityInput>
     with TickerProviderStateMixin {
   AnimationController textInputAnimationController;
-  Animation fromAnimation;
-  Animation toAnimation1;
-  Animation toAnimation2;
-  Animation passengersAnimation;
-  Animation departureAnimation;
-  Animation arrivalAnimation;
 
   @override
   void initState() {
     super.initState();
     textInputAnimationController = new AnimationController(
         vsync: this, duration: Duration(milliseconds: 800));
-    fromAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.0, 0.2, curve: Curves.linear));
-    toAnimation1 = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.15, 0.35, curve: Curves.linear));
-    toAnimation2 = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.3, 0.5, curve: Curves.linear));
-    passengersAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.45, 0.65, curve: Curves.linear));
-    departureAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.6, 0.8, curve: Curves.linear));
-    arrivalAnimation = new CurvedAnimation(
-        parent: textInputAnimationController,
-        curve: Interval(0.75, 0.95, curve: Curves.linear));
+    textInputAnimationController.forward();
```

Comment:
It's not supposed to run on init. Only on plus click. Could you remove this line and then I will merge, ok? :)

Reply:
I guess I just forgot to remove that line after testing, I'll do it as soon as I can!

Reply:
I've removed line 20 starting the animation. Sorry for the delay.